@here/harp-datasource-protocol/lib/InterpolatedProperty.ts

```diff
@@ -376,3 +380,7 @@ function procesStringEnocodedNumeralInterpolatedProperty(
 
     return needsMask;
 }
+
+function clamp(value: number, min: number, max: number): number {
```

Comment:
No please don't add yet another clamp function. There is one in THREE.Math and one in our MathUtils.
I would prefer THREE.Math.clamp b/c MathUtils should go away at some point.

Reply:
Of course :+1:  I have used it... but we got a libs conflicts, because somebody used Math.abs() in this file, which in turn is not available in THREE.Math :). And Math.clamp() is not available in the other lib ... So... maybe different (not common) inclusion patter would solve it, so we could have direct lib specifier, instead of Math.<something>.

Reply:
Done.